tests/pipeline-all-amps/repo/Dockerfile

```diff
@@ -44,5 +40,14 @@ RUN chgrp -R Alfresco ${TOMCAT_DIR} && \
   chmod -R g+r ${TOMCAT_DIR}/webapps
 
 #Use the alfresco user
-USER alfresco
+#USER alfresco
+
+ENV TOMCAT_DIR=${TOMCAT_DIR}
+
+ENV ALFRESCO_AMPS="alfresco-aos-module alfresco-device-sync-repo alfresco-glacier-connector-repo alfresco-googledrive-repo-enterprise alfresco-s3-connector alfresco-saml-repo alfresco-share-services"
```

Comment:
There is also an alfresco-azure-connector amps which is alternative to s3-connector. Is it possible to to add it?

Reply:
> The concerns about this approach are:
> 
> * The startup time of the repository container will increase, it is already too slow. This will affect the readiness probes.
> * The installation of AMPs can break the startup. Meaning that the container will just die. In k8s it will endlessly restart without any evidence why. This means that the end result of installation (the container with amps installed) needs to be tested prior to be used in one pipeline or anywhere else, but it is not possible as the list of AMPs is a variable here.
>   So I would recommend to think about a different way of doing that.

when testing with the run scripts, unzip execution time was negligible so I am not expecting it affects the probes
probes should already be configured with a max restart anyway and the log would show any issues, the logic executed is very limited and all dependencies are inside so probably this would be a problem only when we pick the other issue I had in mind that is to install from a volume any custom non alfresco AMPs a customer might have

happy to review alternatives if any but for me having a docker image with all the supported Alfresco AMPS and the opportunity to enable the ones you need is already miles better than what we have now where anyone seem to create their own testing docker image with a custom combination of amps

Reply:
> There is also an alfresco-azure-connector amps which is alternative to s3-connector. Is it possible to to add it?
what are the artifacts coordinates?

Reply:
neve mind, I've found them all https://github.com/Alfresco/acs-packaging/blob/master/tests/tas-all-amps/pom.xml#L115-L313